Starcounter.Uniform/ViewModels/FormItemMetadata.cs

```diff
@@ -5,9 +5,21 @@
 
 namespace Starcounter.Uniform.ViewModels
 {
+    /// <summary>
+    /// View-model that provides API to manage form items metadata.
+    /// </summary>
     public class FormItemMetadata : Json
     {
-        internal Dictionary<string, MessageContainer> MessageContainers { get; set; }
+        private Dictionary<string, MessageContainer> MessageContainers { get; set; }
+
+        /// <summary>
+        /// Initializes <see cref="FormItemMetadata"/> with message containers.
+        /// </summary>
+        /// <param name="messageContainers">Message containers</param>
+        public void Init(Dictionary<string, MessageContainer> messageContainers)
```

Comment:
that's not exactly what I meant. Why not this:
```
        public void Init(IEnumerable<string> properties)
        {
            var schema = new TObject();
            _messageContainers = properties.ToDictionary(
                property => property,
                property => new MessageContainer(schema.Add<TObject>(property)));
            Template = schema;
        }
```

Reply:
the difference is that in my version you can't call Init in a way that breaks the FormItemMetadata (maybe if you call it twice? It should probably check if it's already initialized, but it's a nitpick). In the previous version, you have to know how FormItemMetadata works to call Init correctly

Reply:
I totally agree, had the same feelings but misunderstood what you meant apparently. Will fix.